test/language/ignore_carriage_returns.wren

```diff
@@ -1,6 +1,6 @@
-// Sprinkle some carriage returns to ensure they are ignored anywhere:
-System
.print("one"
)
-System.print
("two")
-
-// Generate a compile error so we can test that the line number is correct.
-+ * // expect error
+// Sprinkle some carriage returns to ensure they are ignored anywhere:
+System
.
print(
"one"
)
+System.
print(
"two")
+
+// Generate a compile error so we can test that the line number is correct.
++ * // expect error
```

Comment:
I think this file has been commited by mistake.

Reply:
I believe it used to have carriage returns right before newlines, which confused Python into thinking they were newlines in utils/test.py. Since then I made it open the files as binary to circumvent it, so I suppose it doesn't really make much of a difference.